iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/region/RegionMigrateProcedure.java

```diff
@@ -234,8 +213,10 @@ public void deserialize(ByteBuffer byteBuffer) {
       originalDataNode = ThriftCommonsSerDeUtils.deserializeTDataNodeLocation(byteBuffer);
       destDataNode = ThriftCommonsSerDeUtils.deserializeTDataNodeLocation(byteBuffer);
       consensusGroupId = ThriftCommonsSerDeUtils.deserializeTConsensusGroupId(byteBuffer);
+      coordinatorForAddPeer = ThriftCommonsSerDeUtils.deserializeTDataNodeLocation(byteBuffer);
```

Comment:
consider compatibility

Reply:
The compatibility for UpdateRegionLocationPlan is remained. For RegionMigrateProcedure, seems not necessary to keep compatibility. User should not upgrade IoTDB if any RegionMigrateProcedure is not finished.